ehitchhiking-ui/src/app/shared/services/profile-modal.service.ts

```diff
@@ -0,0 +1,64 @@
+import {UserService} from '@shared/services/user.service';
+import {Injectable} from '@angular/core';
+import {ProfileModalApiService} from './api.services/profile-modal.api.service';
+import {User} from '@shared/models/user';
+import {Car} from '@shared/models/car';
+
+@Injectable()
+export class ProfileModalService {
```

Comment:
This service is excess. Pls move all mappings to ProfileModalApiService + inject UserService in the ProfileModalComponent

Reply:
did it